src/Util/include/JM/jm_vector.h

```diff
@@ -145,20 +145,25 @@ Default definition below is jm_diff and is implemented as (int)(first-second)
 #define jm_diff(first, second) (int)(first-second)
 
 /**
-  \brief jm_vector_find functions use linear search to find items in a vector. JM_COMPAR_OP is used for comparison.
+  \brief Uses linear search to find items in a vector. JM_COMPAR_OP is used for comparison.
+
+  @param a - the vector;
+  @param item - the searched item;
 
-  T* jm_vector_find(T)(jm_vector(T)* a, T item, jm_compare_ft f)
+  Return:
+    T* jm_vector_find(T)(jm_vector(T)* a, T item, jm_compare_ft f)  returns a pointer to the found item or NULL if not found
+*/
+#define jm_vector_find(T) jm_mangle(jm_vector_find, T)
```

Comment:
It's a bit hard to see from the diff, but it looked like this:

```
/**
 * Doc for macro1
 * Doc for macro2
 * More doc for macro1
 * More doc for macro2
 * /
#define macro1
#define macro2
```

So I split it